sql/update.go

```diff
@@ -90,20 +93,68 @@ func (p *planner) Update(n *parser.Update) (planNode, error) {
 		vals = append(vals, val)
 	}
 
+	if len(cols) != len(vals) {
+		panic("columns not equal updated values")
```

Comment:
there's a similar check in `insert.go` but it returns an error rather than panicking.